redhat-developer/app-services-cli
# fix(serviceaccount create): display processing text while creation

### Description

Show a processing text when the service account is being created.

![Screenshot from 2021-03-16 12-51-27](https://user-images.githubusercontent.com/23582438/111294657-8cb40800-8670-11eb-8988-fa3ff217b3e2.png)

Fixes #447 

### Verification Steps

1. Run `go run ./cmd/rhoas serviceaccount create`.
2. Enter the values prompted.
3. After entering the description, the CLI should show `Creating service account "<service-account-name>"`

### Type of change


### Checklist
